apiutils/task.go

```diff
@@ -0,0 +1,21 @@
+package apiutils
```

Comment:
As I said IRL, I would just have copied this function into the dispatcher rather than having a new package.